client/character.lua

```diff
@@ -212,6 +212,7 @@ local function createCharacter(cid)
         cid = cid
     })
     destroyPreviewCam()
+    TriggerServerEvent('qb-cityhall:server:getIDs')
```

Comment:
Is this needed? I thought qbx-cityhall was consuming the character creation event to give ids?